lib/ProductOpener/Ingredients.pm

```diff
@@ -439,7 +439,7 @@ my %the = (
 my @labels = (
 	"en:fair-trade-organic", "en:organic", "en:fair-trade", "en:pgi", "en:pdo", "fr:label-rouge",
 	"en:sustainable-seafood-msc", "en:responsible-aquaculture-asc",
-	"fr:aoc", "en:vegan", "en:vegetarian"
+	"fr:aoc", "en:vegan", "en:vegetarian", "pl:bio",
```

Comment:
It looks like we don't have a Polish translation for the en:organic label in the labels.txt taxonomy. (I'm guessing bio means organic in Polish?).  So instead of adding pl:bio here, it should be added to the labels.txt taxonomy instead.

Reply:
Yes, that's a translated "en:organic", it's added to `labels.txt` in this PR
My understanding was that I should put it here if I want it to be parsed from ingredients list, e.g. with ingredients like "papaja bio". Is that wrong?

Reply:
Here we need only the canonical name of the label you want to be parsed from the ingredients list (en:organic), and all synonyms in all languages will be used.

Reply:
Awesome, I didn't know that. 
Can I add this as a comment here to clarify it for the future? :slightly_smiling_face: